SearchResults.js

```diff
@@ -0,0 +1,147 @@
+class SearchResults {
+  constructor(element) {
+    this.element = element;
+    this.data;
+  }
+
+  async dataFetch() {
```

Comment:
this function should be inside the form class, since the searchResutls class has to handle only functionalities that realtes to parsing the response from the server and manipulating the dom